### ISSUE TITLE
new new release for PlatformIO to update

### ISSUE THREAD

**Issue by jezdd** [HelTecAutomation/CubeCell-Arduino#182]
Can we have a new stable 1.3 release please as the PlatformIO team will no update the framework until this is down - those working on PlatformIO framework are in the crazy position of working on code than is now 4 month of commits behind

**Comment by eikaramba** [comment]
yes +1, i think a lot of people are using platformUI nowadays. appreciate the update

**Comment by wtremmel** [comment]
+1 from me as well

**Comment by rony-vargas** [comment]
+1 Here

**Comment by PeterDorsman** [comment]
Release 1.3.1 
1-7-2021
👍
